photon-client/src/main/scala/com/linkedin/photon/ml/hyperparameter/HyperparameterHelper.scala

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright 2018 LinkedIn Corp. All rights reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linkedin.photon.ml.hyperparameter
+
+import scala.util.parsing.json.JSON
+
+import breeze.linalg.DenseVector
+
+import com.linkedin.photon.ml.HyperparameterTuningMode
+import com.linkedin.photon.ml.util.DoubleRange
+
+
+/**
+ * Helper class to read prior observations and configurations of hyper-parameters, and output the results.
+ */
+object HyperparameterHelper {
```

Comment:
This should have a more specific name. It looks like this module is in charge of deserializing hyperparameter evaluations and configuration, so something like `HyperparameterSerialization` might work.